src/lib/routes/admin-api/metrics.ts

```diff
@@ -74,6 +85,9 @@ class MetricsController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Metrics'],
+                    summary: 'Gets a list of applications',
+                    description:
+                        'Gets all applications created via metrics reporting and manual creation',
```

Comment:
or are there other kinds of applications?

```suggestion
                        'Returns all applications registered with Unleash. Applications can be created via metrics reporting or manual creation',
```